plugin/pkg/global/resourcereferencemanager/admission_test.go

```diff
@@ -684,5 +684,239 @@ var _ = Describe("resourcereferencemanager", func() {
 				})))
 			})
 		})
+
+		Context("CloudProfile - Update Kubernetes versions", func() {
```

Comment:
Can you add a test, that checks that `Admit` succeeds if k8s/machine image versions weren't changed?